docs/handbook/spending-money.md

```diff
@@ -0,0 +1,74 @@
+# Spending money
+
+There are many occassions when you will need to spend company money.
+
+PostHog is a lean organization - the less we spend, the more time we have to make sure the company takes off. However, it is more important you are productive, healthy and happy.
+
+Please just spend company money like it's your own.
+
+If it's a trivial expense, just buy it. We'll provide you with a company card with a $500/month spending limit for this reason.
+
+If it's a larger expense and doesn't fit into the items here, please **raise a policy suggestion for it as a pull request** in this doc, so we can document our decision making into our policy rather than making everything case by case.
+
+## Trivial expenses
+
+Just do it.
+
+This means expenses that are under $75 one off or under $20/month recurring that we can cancel easily.
+
+Make sure you keep copies for all receipts. We *cannot* pay you back for anything without a receipt.
+
+## Making larger purchases
+
+If your purchase fits within the policy below, there is no need to ask. We **cannot** pay you back for anything without a receipt.
+
+You may not have enough space on your company card if you're a new starter, just ask James (and if he's unavailable, Tim) to increase your limit.
+
+## Equipment
+
+PostHog is an all-remote company. This means it's important you have an ergonomic setup at home to be as productive as possible.
+
+PostHog will provide you with office equipment. Please note that it remains PostHog's property.
+
+### Laptop, monitor and accessories
+
+We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you.
+
+Our general laptop of choice is a MacBook Pro. Please look for a recent refurbished model. We'd prefer everyone uses Apple laptops, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier not many.
+
+The main thing to look for is that you have enough RAM. 8GB is (frustratingly!) the minimum you'll realistically need if you're browsing the web, writing code and using Slack.
```

Comment:
I'd say 16GB for developers, 8GB for everyone else.

Reply:
And even more if you're doing devops :D 

Pictured below a typical usage when I'm running a bunch of containers via k8s/helm.

![image](https://user-images.githubusercontent.com/53387/79119963-ec839e00-7d91-11ea-8a1d-8324acd52c70.png)

Reply:
this is terrifying.